src/MSBuild/app.config

```diff
@@ -114,14 +114,14 @@
         </dependentAssembly>
         <dependentAssembly>
           <assemblyIdentity name="System.Threading.Tasks.Extensions" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
-          <bindingRedirect oldVersion="0.0.0.0-4.2.1.0" newVersion="4.2.1.0" />
+          <bindingRedirect oldVersion="0.0.0.0-4.2.3.0" newVersion="4.2.3.0" />
         </dependentAssembly>
         <dependentAssembly>
           <assemblyIdentity name="System.ValueTuple" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
           <!-- It is unusual to redirect down, but in this case it's ok: 4.0.3.0 forwards
                to 4.0.0.0 in the GAC, so this just removes the need to redistribute a file
                and makes that resolution faster. -->
```

Comment:
Just checking: is this property still true for ValueTuple 4.0.5.0?

Reply:
System.ValueTuple contains an implementation for net462 and net47. >= net471 has a placeholder file so that the inbox (GAC) assembly will be used.